src/App.js

```diff
@@ -1,26 +1,69 @@
 import React from 'react';
 import logo from './tcl-logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  useRouteMatch,
+} from 'react-router-dom';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>TCL-16</h1>
-        <p>
-          Welcome to TCL-16! This app component is a good starting place for the
-          Smart Shopping List!
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <Router>
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1>TCL-16</h1>
+          <p>
+            Welcome to TCL-16! This app component is a good starting place for
+            the Smart Shopping List!
+          </p>
+          <h2>Week 1</h2>
+
+          <MenuLink activeWhenExact={true} to="/ListView" label="List View" />
+          <MenuLink to="/AddItem" label="Add Item" />
+
+          <Switch>
+            <Route exact path="/ListView">
+              <ListView />
+            </Route>
+            <Route exact path="/AddItem">
+              <AddItem />
+            </Route>
+          </Switch>
+        </header>
+      </div>
+    </Router>
+  );
+}
+
+function MenuLink({ label, to, activeWhenExact }) {
```

Comment:
I'd recommend setting a default for `activeWhenExact` here so the `useRouteMatch` hook doesn't receive undefined when the optional param isn't sent.

Reply:
```suggestion
function MenuLink({ label, to, activeWhenExact = false }) {
```

I've included a code change you can accept into a commit here for what I was referring to in the comment above.